packages/col/src/col.ts

```diff
@@ -50,9 +50,10 @@ const ElCol = defineComponent({
     },
   },
   setup(props, { slots }) {
-    const gutter = inject('ElRow', 0)
+    const elStatus = inject('ElStatus', { gutter: 0 })
```

Comment:
Why change the key name to `ElStatus`? Please set it back to `ElRow` because this is only for `ElRow`, `ElStatus` will cause syntax ambiguity.

Reply:
Ok